src/releng_docs/index.rst

```diff
@@ -34,7 +34,7 @@ Install Nix:
 
 .. code-block:: bash
 
-    % curl https://raw.githubusercontent.com/mozilla-releng/services/master/nix/setup.sh | sh
+    % curl https://raw.githubusercontent.com/mozilla-releng/services/master/nix/setup.sh | bash
```

Comment:
What if user does not have bash installed :) maybe we should use $SHELL variable.

Anybody knows if there is some standers for this?